fiftyone/utils/sapiens.py

```
@@ -0,0 +1,296 @@
+
+    Sapiens2 pose is top-down: it estimates keypoints inside person boxes.
+    Boxes come from a ``prompt_field`` of :class:`fiftyone.core.labels.Detections`;
+    when none are present the full image is used as a single box, matching the
+    upstream demo's detector fallback.
+    """
+
+    @property
+    def required_keys(self):
+        return ["filepath", "prompt_field"]
+
+    def __call__(self, d):
```

Ruff (0.16.1)

[warning] Missing return type annotation for special method `__call__`

(ANN204)